### DISCUSSION TITLE
how to load a model without installing it?

### DISCUSSION THREAD

**Discussion by DSLituiev** [explosion/spaCy#10084] [Help: Best practices]
I have a custom pipeline that uses factories.
When I package it and try to load, I am getting in all sorts of errors depending on where in the package I point.
e.g. I cannot load the distribution `tar.gz` file

```
OSError: [E053] Could not read meta.json from packages/en_pathology_ner-0.0.3/dist/en_pathology_ner-0.0.3.tar.gz
```

My application is where i would have multiple models dropped in a location on drive, so installing each of them  is not practical.
This has worked so far for models that are just trained checkpoint, but now as I need to include `Language.factory` modifiers and package them it breaks. What are good workarounds here?

**Comment by polm** [comment]
If you extract the `.tar.gz` file you can load the model by passing a path to the directory that contains the `config.cfg` file to `spacy.load`.

>  now as I need to include Language.factory modifiers and package them it breaks.

Can you clarify what you mean by "Language.factory modifiers" and how it breaks? There's probably a way you can package that so it works...
